engine/src/main/java/org/hibernate/search/spatial/impl/CoordinateHelper.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.spatial.impl;
+
+import java.io.IOException;
+
+import org.apache.lucene.index.AtomicReader;
+import org.apache.lucene.index.NumericDocValues;
+
+/**
+ * Utility class containing functions to read the value of a spatial coordinate field from an index and convert it to a
+ * double.
+ *
+ * @author Davide D'Alto
+ */
+public final class CoordinateHelper {
+
+	private CoordinateHelper() {
+	}
+
+	public static NumericDocValues loadCoordinateDocValues(AtomicReader atomicReader, String field) throws IOException {
```

Comment:
right,  initially it was longer but after some clean up it became smaller.
I'll remove it